checkout.md

```diff
@@ -0,0 +1,225 @@
+Note that you may submit a change to a Data Carpentry lesson
+when qualifying as a Software Carpentry instructor:
+the goal is to be sure you are familiar with the process.
+
+Your change can be a new exercise or diagram,
+or improve one or more of the lesson's learning objectives.
+Please note that:
+
+1.  If you want to teach Software Carpentry,
+    you must submit the change as a pull request on GitHub.
+    If you want to teach Data Carpentry,
+    you may submit the change either as a pull request
+    or by filing an issue in GitHub that describes your proposed change.
+    Either way,
+    send a link to the pull request or issue to [{{page.email}}]({{page.email}})
+    so that she can shepherd it through the review process.
+
+2.  Your change doesn't have to be accepted in order for you to become an instructor:
+    the point is to make sure that you understand the process and have met the people who review material.
+
+3.  Be sure to format your submission in the same way as the content that's already there.
+    For example,
+    lessons should include the 'challenge' style,
+    while learning objectives should be in the quoted block at the top of the lesson.
+    The `CONTRIBUTING.md` file in each lesson has guidelines
+    (and improvements to these guidelines are welcome).
+
+4.  The lesson maintainers get many pull requests like this,
+    so please take some time to write a good commit message to help them understand what you're doing.
+    For example:
+
+    ~~~
+    Add a 'commit --amend' exercise
+
+    Everybody will botch a commit message at some point.  This
+    exercise introduces them to an easy way to fix those mistakes.
+    ~~~
+
+    will make life much easier for the maintainer than:
+
+    ~~~
+    Instructor-training assignment
+    ~~~
+
+5.  You only have to submit *one* change to move on to the next step, not one per Carpentry.
+
+6.  We do *not* want exercises or other material that introduce new concepts:
+    our lessons are already full to bursting.
+
+7.  If you are writing an exercise,
+    We strongly prefer contributions to the Data Carpentry lessons right now (since that material is much younger)
+    or to our fledgling "Python for complete novices" lesson.
+    Please include a solution to your exercise;
+    please also make sure the exercise runs on all three platforms (Windows, Mac OS X, and Linux),
+    and try to give it a scientific theme
+    (reversing a string isn't particularly motivational for our audience).
+
+8.  If you would rather do a diagram, we are happy to take submissions to any of our lessons.
+    Please use SVG, since vector formats are easier to manipulate than raster formats.
+
+9.  We also appreciate improvements to our lessons' learning objectives.
+    In particular, we would like checked against the actual lesson content,
+    and would also like them labelled according to the levels of Bloom's Taxonomy.
+
+The lessons you may submit to are:
```

Comment:
Yup - that's why I want the SWC and DC sites to link to this doc, and the follow-up emails we send trainees to point to it, so that there's one authoritative source to update.